### PR TITLE
move ACI guide to Network section with TOC changes

### PR DESCRIPTION
**Pull request by acozine**
##### SUMMARY
Moves ACI guide to Network section of docs. This will put all network-specific resources in one section, avoid future disruption from moves, and (with any luck) encourage more vendors to contribute Vendor Guides.

##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
Network documentation

##### ANSIBLE VERSION
2.5

##### ADDITIONAL INFORMATION
The main nav/TOC could still be improved, but I think this is good for now.

### CODE REVIEW

**Review by dharmabumstead — APPROVED** [review]

**Comment by dharmabumstead** [comment]
Looks good to me. Thanks @acozine!

**Comment by dagwieers** [comment]
Well, I have a problem with calling it "Vendor guides", because it limits the scope of what we're talking about. Besides the ACI guide is not a vendor guide. If there'll be a Meraki guide it may also not be a vendor guide. If it would be, you'd be probably having te vendor vetting every change disallowing contributions and I don't think we want to go there...

My other concern is moving this when there is only one guide. If we would have a handful of guides, making a new section for those makes sense. But currently it is unclear if there will be more guides, so as it is I am not convinced moving it at this time is useful. I think it currently is better placed in the Scenario Guides when I think about it.

Moving in itself is not such a big deal as we can have redirects in place (as we already do for moved stuff).

**Comment by acozine** [comment]
@dagwieers "vendor_guides" is in the directory tree, it is not in the navigation. If you generate the nav, you'll see that the Network ACI Guide appears at the top level under "Ansible Network" in the left nav and on the main page of the Network section.

I'm interested in the terminology, though - what term would you use in place of "vendor guide"?

**Comment by dagwieers** [comment]
@acozine Directory-structure is as important, because visible from URL. But I'm not worried about that, it's easy to fix.

We could name it for the intended purpose **"Starter Guides"** or name it after what it is about **"Product Guides"** or **"Technology Guides"** (which doesn't say a lot). **Platform Guides** now is already taken ;-)

**"Scenario Guides"** I liked as well, because you could look at it from different ways, not necessarily bound to a single technology, it could also include stuff about methodology, specific use-cases, managing heterogenous infra or general practices. That's why I am not 100% convinced we need to move *everything* Network-related together.

**Comment by acozine** [comment]
@dagwieers I'm fine with calling the directory "Platform Guides" - the short-form technical platform docs (http://docs.ansible.com/ansible/devel/network/user_guide/platform_index.html) are referred to as "Platform Details", and they live in the User Guide section. Updated in 95583c050675a98d2ad40cd6c037ec38bdd9ff73. Any objections @dharmabumstead @gundalow?

**Comment by acozine** [comment]
Hm, looking again I think "Product Guides" is probably better, since ACI isn't a platform per se. Latest change is now b74c3f890ae326a878cc04122d4d6f19cea21879. @dagwieers @gundalow @dharmabumstead

File: docs/docsite/rst/network/user_guide/index.rst [hunk 1]
```diff
@@ -13,3 +11,8 @@ If you're new to Ansible, or new to using Ansible for network automation, start
 
 .. toctree::
    :maxdepth: 2
+
+   network_best_practices_2.5
+   network_debug_troubleshooting
+   network_working_with_command_output
+   platform_index
```
Comment (gundalow) [inline docs/docsite/rst/network/user_guide/index.rst:18]: Missing newline.

**Review by gundalow — APPROVED** [review]

File: docs/docsite/rst/network/user_guide/index.rst (hunk 1 above)
Comment (acozine) [reply docs/docsite/rst/network/user_guide/index.rst:18]: fixed in c9b46e61126d652a8e4f3c8077dce43caefb2285

**Review by gundalow — APPROVED** [review]

**Comment by acozine** [comment]
Withdrawing for now - we'll revisit this when we get additional guides for network stuff.

**Pull request closed without merging**